### ISSUE TITLE
Add one filter to change capabilities for view and edit of entries

### ISSUE THREAD

**Issue by Shelob9** [CalderaWP/Caldera-Forms#2466]
https://wordpress.org/support/topic/form-editing-capability-for-editors/

**Comment by deepvinsoft** [comment]
anyone help me to This change capability for view and edit of entries? Admin login detail's and login site frontend means entries list popup show and edit and update. but other role using login means not show this popup and not show edit option with frontend.

**Comment by iouston** [comment]
Hi,
Do you have any news about this feature? 
I have the same need and it's quite urgent...
Thank's by advance

**Comment by deepvinsoft** [comment]
yes i got a solution on this. if you need please come to skype : DeepVinSoft

**Comment by ShawnRisk** [comment]
Has there been any updates on this feature?

**Comment by deepvinsoft** [comment]
yes, I have the solution. can you message on skype? my id : DeepVinSoft
